ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -19,6 +19,30 @@ import UIKit
 @objc(MSFBottomCommandingController)
 open class BottomCommandingController: UIViewController {
 
+    /// View controller that will be displayed below the bottom commanding UI.
+    @objc public var contentViewController: UIViewController? {
+        didSet {
+            if isViewLoaded,
+               oldValue != contentViewController {
```

Comment:
Would this be a little more idiomatic with a guard like this?
```
guard isViewLoaded, oldValue != contentViewController else { return }
```

I'm not 100% sure I'm connecting the braces properly so I might be missing something.

Reply:
Thanks for the swifty suggestions!